fee_allocator/accounting/distribution.py

```diff
@@ -34,28 +36,13 @@ def calc_and_split_incentives(
         pool_share = pool_fees / Decimal(total_fees)
         # If aura incentives is less than 500 USDC, we pay all incentives to balancer
         total_incentive = pool_share * fees_to_distr_wo_dao_vebal
-        aura_incentives = round(total_incentive * aura_vebal_share, 2)
-        # If pool has already existing X aura incentives, then it gets precise split of incentives between aura and bal
-        # as aura_bal_ratio
-        cumulative_aura_incentives = Decimal(0)
-        for aura_brib in existing_aura_bribs:
-            if aura_brib['proposal'] == mapped_pools_info.get(pool, "N/A").lower():
-                # Calculate cumulative aura incentives for this pool
-                cumulative_aura_incentives = Decimal(sum([x['value'] for x in aura_brib['bribes']]))
-        # If cumulative aura incentives are more than X USDC, we distribute precisely between aura and bal
-        if cumulative_aura_incentives >= min_existing_aura_incentive:
-            print(f'Pool {pool} has {cumulative_aura_incentives} aura incentives! Allocating precisely...')
-            bal_incentives = round(total_incentive - aura_incentives, 2)
-        else:
-            if aura_incentives <= min_aura_incentive:
-                aura_incentives = Decimal(0)
-                bal_incentives = round(total_incentive, 2)
-            else:
-                # All goes to aura in this case
-                aura_incentives = round(total_incentive, 2)
-                bal_incentives = Decimal(0)
-        fees_to_dao = round(pool_share * fees_to_distribute * dao_share, 2)
-        fees_to_vebal = round(pool_share * fees_to_distribute * vebal_share, 2)
+        aura_incentives = round(total_incentive * aura_vebal_share, 4)
```

Comment:
Change rounding to 4 decimal places, seems to fix our recon problems.

Reply:
Interesting. Finally a fix though!